dist/dropdown/ds6/dropdown.css

```diff
@@ -82,13 +93,13 @@ div.menu__item[role^="menuitem"]:hover,
 div.listbox__option[role^="option"]:hover,
 a.fake-menu__item:hover,
```

Comment:
Do we need this background color for `:focus` too?

Reply:
If you expand the lines right above this, it's all part of the same group of selectors (focus and hover).

Reply:
My bad. Of course the items themselves never gain focus, so I guess this will come from aria-selected instead?

Reply:
Correct.

Reply:
FYI, I moved this around a tiny bit, in order to fix the linting complaining, but it is accomplished exactly the same.